elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -0,0 +1,193 @@
+            organization_extra = OrganizationExtra.objects.create(
+                base=organization,
+                slug=org_slug
+            )
+        if not election_model.organization:
+            election_model.organization = organization_extra.base
+            election_model.save()
+        return organization_extra
+
+    def add_area(self, area_info, election, organization_extra):
+        if area_info['organisation']['territory_code'] == "NIR":
+            partyset_name = "Northern Ireland"
+            country = "ni"
+        else:
+            partyset_name = "Great Britain"
+            country = "gb"
+
+        party_set, _ = PartySet.objects.get_or_create(
+            slug=country, defaults={'name': partyset_name}
+        )
+
+        if area_info['division']:
+            area_type_name = area_info['division']['division_type']
+        else:
+            area_type_name = area_info['organisation']['organisation_type']
+
+        area_type, _ = AreaType.objects.update_or_create(
+            name=area_type_name,
+            defaults={'source': 'elections.democracyclub.org.uk'}
+        )
+
+        if not election.area_types.filter(
+                name=area_type.name).exists():
+            election.area_types.add(area_type)
+
+        if area_info['division']:
+            area_name = area_info['division']['name']
+            post_role = area_info['division']['geography_curie']
+            post_extra_slug = ":".join([
+                area_info['division']['division_type'],
+                area_info['division']['geography_curie'].split(':')[-1]
+            ])
+            area_identifier = area_info['division']['geography_curie']
+            winner_count = area_info['division']['seats_total']
+        else:
+            # TODO
+            area_name = area_info['organisation']['official_name']
+            post_role = area_info['elected_role']
+            post_extra_slug = area_info['organisation']['slug']
+            area_identifier = ":".join([
+                area_info['organisation']['organisation_type'],
+                area_info['organisation']['slug'],
+            ])
+            winner_count = 1
+
+        area, _ = Area.objects.update_or_create(
+            identifier=area_identifier,
+            defaults={
+                'name': area_name,
+                # 'parent': parent_area,
+            }
+        )
+
+        AreaExtra.objects.get_or_create(
+            base=area,
+            defaults={'type': area_type}
+        )
+
+        post, _ = Post.objects.update_or_create(
+            organization=organization_extra.base,
+            area=area,
+            defaults={
+                'role': post_role,
+                'label': area_name,
+            })
+
+        post_extra, _ = PostExtra.objects.update_or_create(
+            slug=post_extra_slug,
+            defaults={
+                'base': post,
+                'party_set': party_set,
```

Comment:
I think the `group` should be set here as well - at the moment it should be one of `Scotland`, `England`, `Wales` or `Northern Ireland`.